src/index.css

```diff
@@ -195,14 +200,21 @@ code {
 
 .run-attributes {
   display: flex;
+  flex-wrap: nowrap;
   flex-direction: column;
   width: 100%;
   margin: 0 0 0 1rem;
+  overflow: clip;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
Instead of repeating the properties in several places you could take advantage of css and create a class for this (`truncate` is customary) and reuse it whnever you need a label truncated.